src/commcare_cloud/environment/paths.py

```diff
@@ -1,14 +1,30 @@
 import os
 import sys
+from distutils.sysconfig import get_python_lib
 
 import yaml
 from memoized import memoized_property, memoized
 
-REPO_BASE = os.path.expanduser('~/.commcare-cloud/repo')
-ANSIBLE_DIR = os.path.join(REPO_BASE, 'ansible')
-ENVIRONMENTS_DIR = os.environ.get('COMMCARE_CLOUD_ENVIRONMENTS', os.path.join(REPO_BASE, 'environments'))
-FAB_DIR = os.path.join(REPO_BASE, 'fab')
-FABFILE = os.path.join(REPO_BASE, 'fabfile.py')
+
+def get_virtualenv_bin_path():
+    """
+    Get the bin directory that the current executable is running from
+
+    This is meant to work even when the user isn't inside a virtualenv,
+    but is directly running an executable that lives in the virtualenv bin,
+    so `os.environ["VIRTUAL_ENV"]` does not work here.
+
+    """
+    return os.path.dirname(sys.executable)
+
+
+PACKAGE_BASE = os.path.realpath(os.path.join(os.path.dirname(__file__), '..'))
+ANSIBLE_ROLES_PATH = os.path.realpath(os.path.join(get_python_lib(), '.ansible/roles'))
```

Comment:
@dannyroberts can you explain why we chose to put the roles path in the virtual env?

Reply:
I think it was under this idea
> This PR goes a long way to making it so others can pip install commcare-cloud—possibly everything except actually uploading it to pypi.
that we should package it in such a way that we could eventually push it to pypi.

Imagining commcare-cloud were in pypi, and were installed via
```
pip install commcare-cloud
```
we would then want to be able to run `manage-commcare-cloud install` and `manage-commcare-cloud configure` and have the effect of those things be contained to the virtualenv directory as well (since we would no longer have a commcare-cloud source directory to play with).